### DISCUSSION TITLE
How to install dependencies?

### DISCUSSION THREAD

**Discussion by lakinchris** [libvips/libvips#2337] [Q&A]
Apologies for my incompetence, but I can't tell from the installation instructions how to install dependencies like libjpeg and libpng. What commands do I enter? I have already installed libvips on macOS.

**Comment by jcupitt** [comment]
Hello @lakinchris,

I suppose you are using homebrew, is that right? Just do:

```
brew install vips
```

And all the dependencies (and libvips itself) will be installed automatically.

**Reply by jcupitt** [reply]
Or are you trying to compile something on macOS that uses libvips? You'll need to set up `pkg-config`, see the homebrew docs.

**Reply by lakinchris** [reply]
Oh I see that I have them now with `vips --vips-config`, ty! I couldn't find in the log that they had been installed though.
